game/theater/controlpoint.py

```diff
@@ -59,6 +59,9 @@
     from game import Game
     from gen.flights.flight import FlightType
 
+MIN_LIMIT_FRONTLINE_UNITS: int = 12
+AMMO_DEPOT_FRONTLINE_UNIT_CONTRIBUTION: int = 6
```

Comment:
Ah I thought the old limit was 30.  The default ammo objective spawns 3 buildings.

Reply:
It was 30 in 2.5. It was increased to 50 because that includes everything in-transit.

I suppose additional fidelity is probably good for the designer? Maybe the template should spawn just the one building and the campaign designer just places 6 at the base if they want the current behavior? I'm not actually sure what these structures look like, so idk if it makes sense to have it reduced to a single building. If we want to keep the higher fidelity but also the same building layout, could drop this to 2 instead. That way mission designers can change the front line size in increments of 6 but it gets degraded in increments of 2.

Apologies for the rambling ideas without solid opinions 😆 wdyt?

Reply:
![Screen_210527_202420](https://user-images.githubusercontent.com/74509817/119920461-9fc53d00-bf29-11eb-988e-67e291588212.png)

These are the traditional ammo depots.  One `.Ammunition Depot` and two `Hangar B`s.

My fear of moving away from the above template is that `.Ammunition Depots` require two 2000 lbs bombs in order to take out.  In modern scenarios that's not so bad, but what do you do when you have a faction that doesn't have the bomb power?  Full control is probably the best way to go, but I don't know the answer to the above question.

Reply:
After fooling around a little bit I realized the last point doesn't matter.  WWII factions get `.Ammunition Depot` anyway.

Reply:
The change to 12 you made seems reasonable to me. Each base will need one of these to maintain its old supply limit, but that's sort of the point :) (and certainly better than the six I was originally worried about)

It's probably worth adding to the settings at some point.